public/serverScripts.js

```diff
@@ -58,3 +58,17 @@ async function calc() {
   document.getElementById('answer').innerHTML = `= ${json.answer}`;
   document.getElementById('resultInput').value = '';
 }
+
+document.addEventListener('DOMContentLoaded', function(){
+    var input = document.getElementById('resultInput');
+
+  // Execute a function when the user releases a key on the keyboard
+  input.addEventListener("keyup", function(event) {
```

Comment:
from linter, single should be with single quotes,
and same as previously, arrow function prefered over unammed functions

Reply:
ok